services/xcode-select-service.ts

```diff
@@ -30,8 +30,12 @@ export class XcodeSelectService implements IXcodeSelectService {
 	@cache()
 	public async getXcodeVersion(): Promise<IVersionData> {
 		let sysInfoBase = this.$injector.resolve("sysInfoBase");
-		let xcodeVer = await sysInfoBase.getXCodeVersion(),
-			xcodeVersionMatch = xcodeVer.match(/Xcode (.*)/),
+		let xcodeVer = await sysInfoBase.getXCodeVersion();
+		if (!xcodeVer) {
+			this.$errors.fail("xcodebuild execution failed. Make sure that you have latest Xcode and tools installed.");
```

Comment:
I originally made it to failWithoutHelp, but for consistency with https://github.com/NativeScript/nativescript-cli/blob/1955028ac3ded6a012813476f08d0f4f0d89dddd/lib/services/ios-project-service.ts#L1190 i left it like this. If you insist I have no objection of making it fail without help.